apps/sports-helsinki/src/domain/unifiedSearch/useUnifiedSearchVariables.ts

```diff
@@ -90,9 +90,9 @@ export default function useUnifiedSearchVariables(
     filters: {
       q,
       // By default filter by the sports dept. ontology tree id
-      ontologyTreeIds = [SPORTS_DEPARTMENT_ONTOLOGY_TREE_ID],
+      ontologyTreeIds,
       // Limit results inside Helsinki when there is no administrative division(s) selected
```

Comment:
I think these comments should be now removed from here as well 🙂

Reply:
moved to proper place, those comments are still important to have